tests/test_declarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -93,22 +93,26 @@ it(\`handles
   console.log("hello!");
 });
 
-test("does something really long and complicated so I have to write a very long name for the test", done => {
+test("does something really long and complicated so I have to write a very long name for the test", (
+  done
```

Comment:
this seems like a clear regression, not sure what the behavior is meant to be testing... probably worth looking into, as I imagine the intent is for this to all stay on one line.